src/js/edu-benefits/config/migrations.js

```diff
@@ -0,0 +1,15 @@
+import _ from 'lodash/fp';
+
+export function urlMigration(formId) {
+  // 0 -> 1, we have split the edu bundle into form-specific bundles with a new urlPrefix
+  // and this replaces the saved returnUrl if necessary
+  return (savedData) => {
+    const savedReturnUrl = _.get('return_url', savedData.metadata);
```

Comment:
It should be `returnUrl`.

Reply:
The comment here is misleading, `return_url` is correct, I have updated the comment to match, and have also renamed the parameter `urlPrefix` (rather than `formId`, which is also misleading).